runtime/gen_primitives.sed

```diff
@@ -15,22 +13,13 @@
 #*                                                                        *
 #**************************************************************************
 
-# duplicated from $(ROOTDIR)/runtime/Makefile
+/^CAMLprim value/ {
+  :loop
+  /)/ ! {
+    N; b loop
+  }
+
+  s/\n  */ /g
```

Comment:
I didn't know you had conditional jumps in sed...  This seems to be in POSIX sed, even.  

This said, if you're willing to suppose primitive definitions take at most 2 lines, there's a loop-free approach:
on seeing `/^CAMLprim /`, fetch next line with `N`, then remove everything following the first `)`.

Reply:
I could, but `caml_array_fill` would disagree!
https://github.com/ocaml/ocaml/blob/ac52c3ab4a545d77eea034993a0947ba5e79762b/runtime/array.c#L557-L560
I think there might be fewer surprises (apart from the script itself) if we loop in sed.